MELA/makefile

```diff
@@ -28,6 +28,10 @@ MELAPKGPATH = $(MELADIR)/
 LIB = libJHUGenMELAMELA.so
 LIBRULE = $(MELALIBDIR)/$(LIB)
 
+pyLIB = Mela$(shell python3-config --extension-suffix)
```

Comment:
python-config/python3-config requires python-dev package. You might want to implement an alternative name specification.

Reply:
python-dev is a dependency of pybind11 [see here](https://pybind11.readthedocs.io/en/stable/basics.html#linux-macos)

If the person doesn't have pybind11 installed then this is a moot point. I can change the python check in setup.sh to instead look for an installation of pybind11